### ISSUE TITLE
feat: allow query without ordering of distance

### ISSUE THREAD

**Issue by eikooc** [codediodeio/geofirex#15]
It seems like it is only possible to query with ascending geohashes [link to queryPoint](https://github.com/codediodeio/geofirex/blob/master/src/collection.ts#L163).
It would be nice not being forced to have that ordering, instead it could be an optionally supplied ordering.

**Comment by ajain-1** [comment]
@mjpoulsen does your forked library have the ability to order by other parameters like a field in the same document?

**Comment by mjpoulsen** [comment]
> @mjpoulsen does your forked library have the ability to order by other parameters like a field in the same document?

No, I only forked this repo to fix several errors I was encountering with my project.

**Comment by ajain-1** [comment]
Ok, thank you for the prompt reply!
